### ISSUE TITLE
4.4.2 CTD on Next Turn

### ISSUE THREAD

**Issue by ausberg** [LoneGazebo/Community-Patch-DLL#10636]
_1. Mod Version (X.Y.Z). Current Version: 4.4.2_
4.4.2 all components

_2. DirectX Version_
DX 10 & 11

_3. List of Other Mods_
More Unique Components for VP (v 87)

_4. Describe the Issue_
The game crashes to desktop when advancing to the next turn, after Napolean asks for friendship.

______________________________________________________________________________________
_5. Save Game From 1 Turn Before (ALWAYS ATTACH THIS IF POSSIBLE)_
[Nebuchadnezzar II_0335 AD-1915.zip](https://github.com/LoneGazebo/Community-Patch-DLL/files/14158865/Nebuchadnezzar.II_0335.AD-1915.zip)

_6. Logs (ALWAYS ATTACH THESE IF POSSIBLE)_
[Logs.zip](https://github.com/LoneGazebo/Community-Patch-DLL/files/14158872/Logs.zip)

_7. CvMiniDump.dmp File (ATTACH IF REPORTING A GAME CRASH)_
[CvMiniDump.zip](https://github.com/LoneGazebo/Community-Patch-DLL/files/14158886/CvMiniDump.zip)

_8. Steps to reproduce the Issue (Optional)_
Just end the turn and respond to Napolean's offer of friendship. I tested an alternative path where I'm still at war with Napolean, so I know that isn't the actual issue.

_9. Screenshots of the Issue (Optional)_

**Comment by ausberg** [comment]
Note: I've been trying to play around with the IGE mod, deleting civs and city states but nothing seems to solve the problem. I can't determine anything by reading the logs either.

**Comment by rambo919** [comment]
Got the same problem with a previous game.... probably an event that's about to happen or some AI taking the wrong city or something. No clues in any of the logs and only at that point in the game.

**Comment by rambo919** [comment]
Duplicate of https://github.com/LoneGazebo/Community-Patch-DLL/issues/10626

**Comment by axatin** [comment]
An AI operation has an invalid muster plot. @ilteroi Can you have a look?
![ctd](https://github.com/LoneGazebo/Community-Patch-DLL/assets/95587882/1ff876c9-031f-493d-8379-6cf8a09873c2)
@ausberg Do you still have a save game from before turn 331? Maybe we need it

**Comment by axatin** [comment]
> Duplicate of #10626

Not necessarily, these two could have different causes. In general, always report all reproducible CTDs on github.

**Comment by ausberg** [comment]
> @ausberg Do you still have a save game from before turn 331? Maybe we need it

Luckily I do still have the previous 5 autosaves.

[autosaves.zip](https://github.com/LoneGazebo/Community-Patch-DLL/files/14169008/autosaves.zip)
